### PR TITLE
Update/docs

### PR DESCRIPTION
**Pull request by mattheww95**
Updating docs and some typos in parameters. I am making the PR to dev as I changed the spelling of some parameters and I want to perform additional testing to make sure everything works.

### CODE REVIEW

File: docs/usage/tool_params.md [hunk 1]
```diff
@@ -179,7 +184,7 @@ Kraken2 can be used a substitute for mash in speciation of samples, and it is us
     - headers: A list of headers in the Kraken2 report.  Do not alter this field unless doing pipeline development.
 
 ### Seven Gene MLST
-Run Torstein Tseemans seven gene MLST program.
+Run Torstein Seemans seven gene MLST program.
```
Comment (dpbastedo) [inline docs/usage/tool_params.md:187]:
```suggestion
Run Torsten Seemann's seven gene MLST program.
```

File: docs/usage/usage.md [hunk 2]
```diff
@@ -100,6 +108,51 @@ Numerous steps within mikrokondo can be turned off without compromising the stab
...
+- `--skip_mlst`: Skip seven gene MLST.
+
+#### Datasets
+Different databases/pre-computed files are required for usage within mikrokondo. These can be downloaded or created by the user, and if not configured within the `nextflow.config` file they can be passed in as files with the following command-line arguments.
+
+- `--dehosting_idx`: The minimap2 index to be used for dehosting.
+- `--mash_sketch`: The mash sketch to be used for contamination detection and speciation.
+- `--bakta_db`: Bakta database for genome annotation.
+- `--kraken2_db`: Kraken2 database that can be used for speciation and binning of meta-genomically assembled contigs.
+- `--staramr_db`: An optionaly StarAMR database to be passed in, it is recommended to use the database packaged in the container.
```
Comment (dpbastedo) [inline docs/usage/usage.md:123]:
```suggestion
- `--staramr_db`: An optional StarAMR database to be passed in, it is recommended to use the database packaged in the container.
```

File: docs/usage/usage.md [hunk 3]
```diff
@@ -100,6 +108,51 @@ Numerous steps within mikrokondo can be turned off without compromising the stab
...
+
+#### FastP Arguments
+For simplicity parameters affecting FastP have been moved to the top level. Each argument matches one listed within the [FastP](https://phac-nml.github.io/mikrokondo/usage/tool_params/#fastp) usage section with only a `fp_` being appended to the front of the argument. For a more detailed description of what each argument does please review the tool specific parameters for [FastP](https://phac-nml.github.io/mikrokondo/usage/tool_params/#fastp) here.
+
+- `--fp_average_quality`: If a read/read-pair quality is less than this value it is discarded
+- `--fp_cut_tail_mean_quality`: the quality to trim reads too
```
Comment (dpbastedo) [inline docs/usage/usage.md:129]:
```suggestion
- `--fp_cut_tail_mean_quality`: the quality threshold to trim reads to
```

**Review by dpbastedo — APPROVED** [review]
I have added a few additional comments regarding (very minor) typos or suggested re-wording, but all changes look good to me.

**Comment by mattheww95** [comment]
Merging branch so that parameters with their fixed spelling are merged.

**Review by dpbastedo — COMMENTED** [review]
Looks good!

**Review by dpbastedo — COMMENTED** [review]
Looks good!

**Pull request merged**